geunwoong/WeekAssign02/WeekAssign02/UtilFunction/ExecuteMenu.swift

```diff
@@ -0,0 +1,20 @@
+/*
+ File: ExecuteMenu.swift
+ Purpose: 입력받은 메뉴에 따른 메뉴 실행
+ 
+ Data
+ - menu: Int // 입력받은 메뉴 값의 Int형, 1~6은 정상적인 입력으로 메뉴 실행, -1은 오류상태
+
+ etc
+ - 지금은 -1이 들어오면 아무것도 안하는데 나중엔 먼가 처리를 해주면 좋을듯
+*/
+extension ReflectionSystem {
+    func executeMenu(menu: Int) {
+        if menu == 1 { addReflection() }
+        if menu == 2 { searchReflection() }
+        if menu == 3 { editReflection() }
+        if menu == 4 { deleteReflection() }
+        if menu == 5 { printAllReflection() }
+        if menu == 6 { exitProgram() }
```

Comment:
메뉴를 열거형으로 만들어서 Switch 문을 활용하면 좋을 것 같습니다 !
```suggestion
enum MenuOption: Int {
  case addReflection = 1
  case searchReflection
  case editReflection
  case deleteReflection
  case printAllReflection
  case exitProgram
}

switch menu {
    case .addReflection:
      addReflection()
    case . searchReflection:
      searchReflection()
    case .editReflection:
      editReflection()
    case .deleteReflection:
      deleteReflection()
    case . printAllReflection:
      printAllReflection()
    case . exitProgram:
      exitProgram()
 }
```

Reply:
enum을 사용하면 좋지만, enum을 사용할 필요성을 못 느끼신다면 적어도 If문보다는 switch-case문이 유지보수에도 좋은 구조일 것 같아요!